src/modules/Bootloader.rb

```diff
@@ -204,7 +204,8 @@ def Summary(simple_mode: false)
       # kokso: additional warning that root partition is nfs type -> bootloader will not be installed
       if BootStorage.boot_filesystem.is?(:nfs)
         log.info "Bootloader::Summary() -> Boot partition is nfs type, bootloader will not be installed."
-        return _("The boot partition is of type NFS. Bootloader cannot be installed.")
+        msg = _("The boot partition is of type NFS. Bootloader cannot be installed.")
+        return (simple_mode ? [msg] : [Yast::HTML.Colorize(msg, "red")])
```

Comment:
why having it red? It is not warning. It is correct proposal for NFS.

Reply:
I agree with no highlight but definitely should be shown. I just used the same warning was defined in the NoneBootloader [Summary](https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/none_bootloader.rb#L11)